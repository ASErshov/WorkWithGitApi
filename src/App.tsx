import React from 'react';
import logo from './logo.svg';
import './App.css';
import Header from './components/Header';

function App() {
  return (
    <Header>Добро пожаловать</Header>
  );
}

export default App;
