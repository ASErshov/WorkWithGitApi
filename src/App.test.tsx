import React from 'react'
import renderer from 'react-test-renderer'

import App from './App'

describe('<App />', () => {
  const Component = (
    <App/>
  )

  it('Snapshot is correct', () => {
    const wrapper = renderer.create(Component).toJSON()

    expect(wrapper).toMatchSnapshot()
  })

})
