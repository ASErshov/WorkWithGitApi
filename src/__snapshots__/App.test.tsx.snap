// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Snapshot is correct 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionRelative MuiAppBar-colorPrimary appBar MuiPaper-elevation4"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular toolbar MuiToolbar-gutters"
  >
    <h6
      className="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap"
    >
      Добро пожаловать
    </h6>
  </div>
</header>
`;
